.github/workflows/changelog.yml

```diff
@@ -0,0 +1,33 @@
+# This action requires that any PR targeting the main branch should touch at
+# least one CHANGELOG file. If a CHANGELOG entry is not required, add the "Skip
+# Changelog" label to disable this action.
+
+name: changelog
```

Comment:
Tiniest of the tiny nits: capitalize this since all others start with a capital letter.